### PR TITLE
Refactored Handler Code to Reduce Duplication and add App Presenter

### PR DESCRIPTION
**Pull request by gnovv**
Co-authored-by: Akira Wong <wakira@vmware.com>

### CODE REVIEW

File: cfshim/handlers/presenter_types.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+		Metadata: CFAPIMetadata{
+			Labels:      map[string]string{},
+			Annotations: map[string]string{},
+		},
+	}
+}
+
+
+
+
```
Comment (heycait) [inline cfshim/handlers/presenter_types.go:95]: The app presenter is a little hard to follow since the followup structs aren't sequentially but otherwise lgtm.

Also, probably don't want a bunch of new lines. Can instead make the comment separating packages be more prominent.

**Review by heycait — APPROVED** [review]

File: cfshim/handlers/presenter_types.go (hunk 1 above)
Comment (gnovv) [reply cfshim/handlers/presenter_types.go]: Thank you!

I reordered the structs and added a flowerbox to the separation comment!
